libraries/AP_HAL_Linux/RCInput_PRU.cpp

```diff
@@ -17,7 +18,8 @@
 #include <unistd.h>
 
 #include "GPIO.h"
-#include "RCInput.h"
+
+#define RCIN_PRUSS_SHAREDRAM_BASE   0x4a312000
```

Comment:
I understand that this helps on avoiding recompilation when RCIN_PRUSS_SHAREDRAM_BASE changes. I think this either should be a separate commit or the commit message should have a note that it has also other minor changes to avoid unnecessary recompilation.